teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher < Person
+  def initialize(age, specialization, name)
```

Comment:
- [ ] kindly make sure you initialize also name = "Unknown", parent_permission = true.
Like Example:-
`def initialize(age, specialization, name = "Unknown", parent_permission = true)`